layouts/_default/baseof.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html class="no-js" lang="{{ .Site.LanguageCode | default "en-us" }}">
+<html class="no-js" lang="{{ .Language.Lang | default "en-us" }}">
```

Comment:
> > The current variable `.Site.LanguageCode` does not correctly set the
> > html tag lang attribute for multilingual sites. Instead **it only uses the
> > default language setting**.
> 
> No. Current master version uses the same value as you've set in the `languageCode`. If you don't set anything, then yes, it uses the default value:

You are absolutely right. I'm very new to Hugo and I misunderstood how the `languageCode` variable worked. In my `config.yaml` I have:

```
languages:
  en:
    contentDir: content/en
    languageName: English
  fi:
    contentDir: content/fi
    languageName: Suomi
```

and I mistakenly assumed the code would have been set with the `en` and `fi` parts there. Thanks for clearing that up.


> > https://gohugo.io/variables/page/ describes the page variable
> > .Language.Lang as the most appropriate alternative.
> 
> Why page variable instead of site variable?
> https://gohugo.io/variables/site/#site-variables-list

Just semantics. When using the `page`'s language instead of the `site`'s language, to me it's more clear which language we are talking about in a multilingual environment.

Site variable's documentation says

> the language code of the current locale

Can "the current locale" sometimes be something else than the page's language? I don't even know. But if can, also in that case the `html` lang should be set to be the language of the main page content, even if for example the navigation menus were for some reason in another language.

What do you think?

Reply:
We should use `.Site.Language.Lang`. Here is the main reason from my side:

1. Consistency. Site variables = common site params;  Page variables = params for specific pages.

I don't see any reason to use more specific param, if more general suits us.

---

> Can "the current locale" sometimes be something else than the page's language?

No.

Reply:
I updated the pull request as per your comment. Sorry for the delay.